backend/celery_worker.py

```
@@ -1053,6 +1139,65 @@ def mark_tasks_overdue() -> int:
+        config = HubSpotConfig.query.order_by(HubSpotConfig.id.desc()).first()
+        if config is None:
+            logger.info("Scheduled engagement sync skipped: no HubSpot config found")
+            return
+
+        # Create an import run for engagements
+        svc = HubSpotImportService()
+        try:
+            runs = svc.start_import(object_types=['engagements'])
+            run_ids = [r.id for r in runs]
+            logger.info("Scheduled engagement sync: started import run_ids=%s", run_ids)
+        except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)